FunctionApp/Middleware/ApplicationBuilderExtensions.cs

```diff
@@ -3,6 +3,7 @@
 using System.Linq;
 using System.Text;
 using System.Threading.Tasks;
+using Microsoft.Azure.Functions.Worker;
```

Comment:
Is this intentional?

Reply:
It was. One of the types moved with the latest update.